images/atomic-registry-quickstart/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/origin
+MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
+
+ADD install.sh run.sh uninstall.sh /container/bin/
+ADD registry-console-template.yaml registry-project-template.json registry-login-template.html \
+    /container/etc/origin/
+
+LABEL name="projectatomic/atomic-registry-quickstart" \
+      vendor="Project Atomic" \
+      url="https://projectatomic.io/registry" \
+      summary="Quickstart image for Atomic Registry" \
+      description="This is a quickstart image to install Atomic Registry on a single host. It is an open source enterprise registry solution based on the Origin project featuring single sign-on (SSO) user experience, a robust web interface and advanced role-based access control (RBAC)." \
+      INSTALL='docker run -it --rm \
```

Comment:
I'm asking why we need the image at all.  What does it give us that a
script can't?